packages/babel-preset-env/test/fixtures/debug/browserslists-last-2-versions-not-ie/stdout.txt

```diff
@@ -0,0 +1,31 @@
+@babel/preset-env: `DEBUG` option
+
+Using targets:
+{
+  "android": "76",
+  "chrome": "77",
+  "edge": "17",
+  "firefox": "68",
+  "ios": "13",
+  "opera": "46",
+  "safari": "12.1",
+  "samsung": "9.2"
+}
+
+Using modules transform: auto
+
+Using plugins:
```

Comment:
The plugins listed here is generated from a sort-of modern browserslist configuration. It can help us prioritize bug fixes in the current transforms. because these plugins are expected to be used for a longer time than other standard transforms.